CONTRIBUTING.rst

```diff
@@ -1,37 +1,214 @@
 Contributing
 ============
 
-Running Tests
--------------
+Instructions for contributors
+-----------------------------
+
+In order to make a clone of the `GitHub <https://github.com/aio-libs/aiobotocore>`_ repo: open the link and press the
+"Fork" button on the upper-right menu of the web page.
+
+I hope everybody knows how to work with git and gitlab nowadays :)
+
+Workflow is pretty straightforward:
+
+  1. Clone the `GitHub <https://github.com/aio-libs/aiobotocore>`_ repo
+
+  2. Make a change
+
+  3. Make sure all tests passed
+
+  4. Make sure the coverage doesn't get worse.
+
+  5. Commit changes to own aiobotocore
+
+  6. Correct message for ``commit``. You need read (`Commit message format`_).
 
-.. _GitHub: https://github.com/aio-libs/aiobotocore
+  7. Correct name for ``branch``. You need read (`Branch name format`_).
 
-Thanks for your interest in contributing to ``aiobotocore``, there are multiple
-ways and places you can contribute.
+  8. Make pull request from github page for your branch against master branch
 
-Fist of all just clone repository::
+  9. Optionally make backport pull Request(s) for landing a bug fix
```

Comment:
we don't have backports yet ;)

Reply:
I now think that this is superfluous, since I cannot exactly remember what exactly I wanted to say here. sorry i will remove.